microsoft-azure/4.0/support/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Supported platforms
+---
+
+The following are the supported platforms for Alfresco Content Connector for Azure 4.0:
+
+| Version | Notes |
+| ------- | ----- |
+| Content Services 23.2.x | |

```

Comment:
It looks like the 4.0 and 5.0 content is switched - `4.0` is compatible with 23.1.x and `5.0` ... with 23.2.x

```suggestion
| Content Services 23.1.x | |

```